charts/common/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v2
 name: common
 description: A common library chart that provides accelleran standardized named templates
 type: library
-version: 0.3.9
+version: 0.4.0
```

Comment:
We never should touch the versions ourselves. Release please should suggest a new release. For that reason we should also split up PRs that would need something added/updated in the common charts. There would be one PR for the common chart and one for the new feature/patch in the actual chart. In that last PR we can only open it up properly after there was a new release created and bumped there.